code/modules/antagonists/eldritch_cult/eldritch_magic.dm

```diff
@@ -52,15 +52,21 @@
 	qdel(src)
 
 /obj/item/melee/touch_attack/mansus_fist/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	mansus_attack(target, user, proximity_flag, click_parameters, FALSE)
```

Comment:
I believe you still want to `. = ..()` at the beginning of *this* one, since it fires some signals.